doc/configuration/network.md

```diff
@@ -20,13 +20,10 @@ p2p:
 
 - `trusted_peers`: (optional) the list of nodes' [multiaddr][multiaddr] to connect to in order to
     bootstrap the p2p topology (and bootstrap our local blockchain);
-- `public_address`: [multiaddr][multiaddr] the address to listen from and accept connection
-    from. This is the public address that will be distributed to other peers
-    of the network that may find interest into participating to the blockchain
-    dissemination with the node;
-- `listen_address`: (optional) [multiaddr][multiaddr] specifies the address the node
-    will listen to to receive p2p connection. Can be left empty and the node will listen
-    to whatever value was given to `public_address`.
+-  public_address: [multiaddr] The address to listen from and accept connections. This is the external public address that will be distributed to other gossip peers on the network.  Public address port must match listen address port.
+-  listen_address:  [multiaddr] Internal address that the node will listen to for 
+    receiving p2p connections.  This connection will typically be port forwarded from 
```

Comment:
I would say you are "typically" running an server with a public IP address, not port forwarding.